stable/test/integration/test_env.pony

```diff
@@ -0,0 +1,217 @@
+use "ponytest"
+use "files"
+use "process"
+use ".."
+
+actor EnvTests is TestList
+  new create(env: Env) => PonyTest(env, this)
+  new make() => None
+
+  fun tag tests(test: PonyTest) =>
+    test(TestEnvNoBundle)
+    test(TestEnvEmptyBundleInSameDir)
+    test(TestEnvBundleInSameDir)
+    test(TestEnvBundleInSameDirWithCall)
+    test(TestEnvBundleInParentDir)
+    test(TestEnvBadBundleInNestedAndValidBundleInParentDir)
+
+class TestEnvNoBundle is UnitTest
+  new iso create() => None
+
+  fun name(): String => "integration.Env(no bundle.json)"
+
+  fun apply(h: TestHelper) =>
+    h.long_test(100_000_000)
+    let tmp =
+      try
+        FilePath.mkdtemp(h.env.root as AmbientAuth, "")?
```

Comment:
❔ The `""` is a bit icky, as it'll create the temp dir in the current working directory -- i.e., the checkout of `pony-stable`. It'll get cleaned up, but it still strikes me as a questionable default. (OTOH, I'm not sure if `/tmp` is much better, but I might be overthinking this. We could also create an empty `tmp` dir in the repo, and use that...)